backtesting/backtesting.py

```diff
@@ -719,6 +731,9 @@ def __init__(self, *, data, cash, commission, margin,
         self.position = Position(self)
         self.closed_trades: List[Trade] = []
 
+    def _commission_func(self, order_size, price):
+        return self._commission_fixed + abs(order_size) * price * self._commission_relative
```

Comment:
Will this cover all sorts of commission cases?

Reply:
The base case is solved I would say.

**Nice to have:**
In cryptocurrency exchanges, your commissions tend to be reduced over a rolling 30-day period if you create enough volume.

It would be handy if you could create a scaling commission feature based on the transaction volume. It would also be handy if it can be split into maker/taker. 

![image](https://user-images.githubusercontent.com/27340033/221440128-f54b12e3-1776-40d9-ba9f-633cc2e92804.png)